app/src/main/java/pm/gnosis/heimdall/di/modules/ApplicationModule.kt

```diff
@@ -198,6 +198,7 @@ class ApplicationModule(private val application: Application) {
             }
         }.build()
 
+
```

Comment:
Could we try to avoid multiple new lines (or have them everywhere, I am fine either way)

Reply:
I am for avoiding. This could be good candidate for a custom lint rule